public/index.html

```diff
@@ -9,7 +9,7 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div
       id="root"
-      data-api="http://localhost:3333/v1/"
+      data-api="https://data-api.energidataservice.dk/v1/"
```

Comment:
@tanvirchahal this shouldn't be hardcoded - we need to find solution to make configurable

Reply:
@anuveyatsu how important is it to put the `data-api` in `public/index.html`? 

I suggest that we can put these variables in a `.env` file and load them directly into `Index.js` component  

Then we can have two files `.env` and `.env.test` which can have separate `data-api` URLs 

If its important to keep the `data-api` in `public/index` , then we can try loading these variables from an external JSON file before loading the application but I am not sure how stable would that be.

Reply:
@tanvirchahal ok, I think we can keep it for now 👍